KerbalStuff/blueprints/api.py

```diff
@@ -583,19 +580,10 @@ def create_mod():
             or len(short_description) > 1000 \
             or len(mod_licence) > 128:
         return {'error': True, 'reason': 'Fields exceed maximum permissible length.'}, 400
-    game = None
-    if game_id:
-        game = Game.query.get(game_id)
-    elif game_short:
-        game = Game.query.filter(Game.short == game_short).first()
-    if not game:
-        return {'error': True, 'reason': 'Game does not exist.'}, 400
-    game_version_id = db.query(GameVersion.id) \
-        .filter(GameVersion.game_id == game.id) \
-        .filter(GameVersion.friendly_version == game_version) \
-        .first()
-    if not game_version_id:
+    game_version = GameVersion.query.get(game_version_id)
+    if not game_version:
         return {'error': True, 'reason': 'Game version does not exist.'}, 400
+    game = game_version.game
```

Comment:
Should we check `game.active` here? It's odd that we have the ability to deactivate a game but then we allow creation of new mods for it.

https://github.com/KSP-SpaceDock/SpaceDock/blob/b7cb750b5b71e497f5d791c1754e55c1ac90ed0c/KerbalStuff/objects.py#L122-L126